slack/src/commonMain/kotlin/work/socialhub/planetlink/slack/action/SlackAction.kt

```
@@ -0,0 +1,652 @@
+        if (team != null) return team
+
+        return try {
+            val response = auth.accessor.slack.team().teamInfo(
+                TeamInfoRequest(auth.accessor.token)
+            )
+
+            if (response.isOk) {
+                team = SlackMapper.team(response)
+            }
+            team
+        } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)